README.md

```diff
@@ -330,7 +210,7 @@ Compose open source has been developed by Feature Labs engineering team. The ope
 
 ## Feature Labs
 <a href="https://www.featurelabs.com/">
-    <img src="http://www.featurelabs.com/wp-content/uploads/2017/12/logo.png" alt="Featuretools" />
+    <img src="https://www.featurelabs.com/wp-content/uploads/2017/12/logo.png" alt="Featuretools" />
 </a>
 
 Compose is an open source project created by Feature Labs. We developed Compose to enable flexible definition of the machine learning task. Read more about our rationale behind automating and developing this stage of the machine learning process here.
```

Comment:
I'm assuming we need to update this to Innovation Labs.

Reply:
@kmax12 thoughts?